### PR TITLE
R4 Observation Labs

### PR DESCRIPTION
**Pull request by Manjushavrk**
Description
----
Added valueDateTime.

Screen Shot 2020-12-16 at 9 07 44 AM
Screen Shot 2020-12-16 at 9 07 27 AM

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Comment by BS062721** [comment]
I don't see valueDateTime in screenshots?!

**Review by fpivonka — APPROVED** [review]

**Comment by Manjushavrk** [comment]
> I don't see valueDateTime in screenshots?!

updated the screenshots Brij.

File: lib/resources/r4/observation.yaml [hunk 1]
```diff
@@ -554,19 +554,23 @@ fields:
...
+      "valueDateTime": '2020-07-04T07:15:00Z'
+    }
   note: |
     <ul>
       <li> Vital Signs support writing observations with `valueQuantity` </li>
     </ul>
 
     <ul>
-      <li> Laboratory support `valueQuantity`, `valueCodeableConcept` and `valueString` </li>
+      <li> Laboratory support `valueQuantity`, `valueCodeableConcept`, `valueString` and `valueDateTime` </li>
```
Comment (BS062721) [inline lib/resources/r4/observation.yaml:567]: are we missing period(.) at the end?

File: lib/resources/r4/observation.yaml [hunk 2]
```diff
@@ -596,9 +600,13 @@ fields:
...
       "valueString": '2'
     }
+  example4: |
+    {
+      "valueDateTime": '2020-07-04T07:15:00Z'
+    }
   note: |
     <ul>
-      <li> Laboratory support `valueQuantity`, `valueCodeableConcept` and `valueString` </li>
+      <li> Laboratory support `valueQuantity`, `valueCodeableConcept`, `valueString` and `valueDateTime` </li>
```
Comment (BS062721) [inline lib/resources/r4/observation.yaml:609]: same here?

**Review by BS062721 — APPROVED** [review]

File: lib/resources/r4/observation.yaml (hunk 1 above)
Comment (Manjushavrk) [reply lib/resources/r4/observation.yaml]: updated : https://github.com/cerner/fhir.cerner.com/commit/ff3fe4a9864edf890ffe8d1182919fee145fcd42#diff-e303aa1b2a83c6a68a12604d4956dcd3c31f4ffe40546bbcc31e2cc6588921dcR567

File: lib/resources/r4/observation.yaml (hunk 2 above)
Comment (Manjushavrk) [reply lib/resources/r4/observation.yaml]: updated: https://github.com/cerner/fhir.cerner.com/commit/ff3fe4a9864edf890ffe8d1182919fee145fcd42#diff-e303aa1b2a83c6a68a12604d4956dcd3c31f4ffe40546bbcc31e2cc6588921dcR609

**Review by jamvar — APPROVED** [review]

**Comment by ebbp123** [comment]
Verified Live: 
Screen Shot 2020-12-17 at 4 18 06 PM

**Pull request merged**